From 82a6de914d0d6ff82333a4b65c81f71335e1f9d0 Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Wed, 4 Dec 2024 01:31:46 +0100
Subject: [PATCH] dt-bindings: clock: Add clock ID for IPQ PCS UNIPHY

IPQ9574 expose 3 PCS UNIPHY that expose all the same clock ID for RX and
TX clock. This is needed to correctly parent the EDMA ports and scale
to the correct frequency to permit correct function of attached PHY.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 include/dt-bindings/clock/qcom,ipq-pcs-uniphy.h | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100644 include/dt-bindings/clock/qcom,ipq-pcs-uniphy.h

--- /dev/null
+++ b/include/dt-bindings/clock/qcom,ipq-pcs-uniphy.h
@@ -0,0 +1,13 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
+
+#ifndef _DT_BINDINGS_CLK_QCOM_IPQ_PCS_UNIPHY_H
+#define _DT_BINDINGS_CLK_QCOM_IPQ_PCS_UNIPHY_H
+
+/* 
+ * IPQ9574 expose 3 PCS UNIPHY that expose all
+ * the same clock ID
+ */
+#define UNIPHY_NSS_RX_CLK		0
+#define UNIPHY_NSS_TX_CLK		1
+
+#endif
